### PR TITLE
Toolbox components order is random.

### PR DESCRIPTION
**Pull request by bierik**
Sort components by title ascending.

### CODE REVIEW

**Comment by bierik** [comment]
Fixes https://github.com/4teamwork/bern.web/issues/186

File: ftw/simplelayout/browser/client/test/toolbox.test.js [hunk 1]
```diff
@@ -72,6 +72,26 @@ suite("Toolbox", function() {
       });
       assert.deepEqual(addedNodes, [1, 2, 4]);
     });
+
+    test("toolbox components are sorted asc", function() {
+      var toolbox = new Toolbox({
+        layouts: [0],
+        components: {
```
Comment (maethu) [inline ftw/simplelayout/browser/client/test/toolbox.test.js]: The components are defined in alphabetical order... for manful testcase you should define them not alphabetically.

Comment (maethu) [inline ftw/simplelayout/browser/client/test/toolbox.test.js]: @bierik :up:

**Comment by bierik** [comment]
@jone updated

File: ftw/simplelayout/browser/client/web/js/app/toolbox/Toolbox.js [hunk 2]
```diff
@@ -16,6 +16,20 @@ define([], function() {
       components: {}
     }, _options || {});
 
+    var replaceUmlauts = function(umlaut) {
+      if(typeof umlaut === "string") {
+        var umlauts = {"ä": "ae", "ü": "ue", "ö": "oe", "ß": "ss" };
+        return umlaut.replace(/[äöü]/g, function($0) {
```
Comment (jone) [inline ftw/simplelayout/browser/client/web/js/app/toolbox/Toolbox.js]:
```js
new RegExp(Object.keys(umlauts).join(''), 'g')
```

File: ftw/simplelayout/browser/client/web/js/app/toolbox/Toolbox.js [hunk 3]
```diff
@@ -16,6 +16,20 @@ define([], function() {
       components: {}
     }, _options || {});
 
+    var replaceUmlauts = function(umlaut) {
+      if(typeof umlaut === "string") {
+        var umlauts = {"ä": "ae", "ü": "ue", "ö": "oe", "ß": "ss" };
```
Comment (jone) [inline ftw/simplelayout/browser/client/web/js/app/toolbox/Toolbox.js]: + Capitals (`ÄÖÜ`)

File: ftw/simplelayout/browser/client/dist/simplelayout.js [hunk 4]
```diff
@@ -1001,8 +1001,12 @@ define('app/toolbox/Toolbox',[], function() {
           </div> \
         </div>");
 
+    var components = $.map(options.components, function(component) { return component; }).sort(function(a, b) {
+      return a.title > b.title;
```
Comment (jone) [inline ftw/simplelayout/browser/client/dist/simplelayout.js]: recompile

**Comment by jone** [comment]
:+1:

**Comment by bierik** [comment]
@jone updated

**Comment by jone** [comment]
:+1:

**Pull request merged**